src/components/Share/style.scss

```diff
@@ -0,0 +1,30 @@
+.share {
```

Comment:
greats tips bro. I have since update the code to introduce font-awesome as @mmahalwy suggested. Please review again and let me know.